commands/music/musictrivia.js

```diff
@@ -92,8 +92,9 @@ module.exports = class MusicTriviaCommand extends Command {
       const dispatcher = connection
         .play(
           ytdl(queue[0].url, {
+            filter: 'audio',
             quality: 'highestaudio',
-            highWaterMark: 1024 * 1024 * 1024
+            highWaterMark: 1 << 25
           })

```

Comment:
red is removed and Green is changed

Reply:
okayy nice thankyou bro, btw ytdl-core is now vers 4.1.4

Reply:
hmm bro, it still stops halfway tho

Reply:
i have changed my youtube api key too

Reply:
<img width="545" alt="Screenshot (220)" src="https://user-images.githubusercontent.com/69320092/101473647-0909c180-3985-11eb-8da4-a1e2095fd55a.png">